test/TickMath.starknet.ts

```diff
@@ -163,16 +163,16 @@ describe('TickMath', () => {
         it('is at most off by 1', async () => {
           const jsResult = new Decimal(ratio.toString()).div(new Decimal(2).pow(96)).pow(2).log(1.0001).floor()
           const result = await tickMath.getTickAtSqrtRatio_4f76c058(new BN(ratio.toString()).toTwos(24).toString())
-          const absDiff = new Decimal(result[0].toNumber()).sub(jsResult).abs()
+          const absDiff = new Decimal(result[0].toString()).sub(jsResult).abs()
           expect(absDiff.toNumber()).to.be.lte(1)
         })
         
         it('ratio is between the tick and tick+1', async () => {
           const tick = await tickMath.getTickAtSqrtRatio_4f76c058(new BN(ratio.toString()).toTwos(24).toString())
```

Comment:
Same here, Since the values are not negative, we can use `getTickAtSqrtRatio_4f76c058(ratio.toString())`